nifi-commons/nifi-site-to-site-client/src/main/java/org/apache/nifi/remote/client/http/HttpClient.java

```diff
@@ -181,7 +181,7 @@ public Transaction createTransaction(final TransferDirection direction) throws I
                 commSession.setUserDn(apiClient.getTrustedPeerDn());
             } catch (final Exception e) {
                 apiClient.close();
-                logger.warn("Penalizing a peer {} due to {}", peer, e.toString());
+                logger.warn("Penalizing peer {}", peer, e);
```

Comment:
This particular change should be reverted due to the potential for excessive stack trace logging.